composables/useSocialShare.ts

```diff
@@ -13,12 +13,12 @@ export default function () {
   const shareOnX = (
     text: string,
     url: string = fullPathShare.value,
-    via: string = 'KodaDot',
+    via: string | null = 'KodaDot',
   ) => {
     const shareUrl = new URL('https://twitter.com/intent/tweet')
-    shareUrl.searchParams.set('text', text)
-    shareUrl.searchParams.set('via', via)
-    shareUrl.searchParams.set('url', url)
+    text && shareUrl.searchParams.set('text', text)
+    via && shareUrl.searchParams.set('via', via)
+    url && shareUrl.searchParams.set('url', url)
```

Comment:
```suggestion
    shareUrl.searchParams.set('text', text)
    via && shareUrl.searchParams.set('via', via)
    shareUrl.searchParams.set('url', url)
```
was maybe mb , I meant only the `via`, even though the other could also be `''` , it's not the intent for those to be nullish, this makes more sense

commit suggestion if you agree , sorry for being picky :/

Reply:
That's fine : )